src/host.cpp

```diff
@@ -1079,26 +1082,24 @@ namespace rhost {
             }
         }
 
-        void setCallbacksWindows(structRstart& rp) {
 #ifdef _WIN32
+        void setCallbacksWindows(structRstart& rp) {
```

Comment:
Sorry, didn't notice it on the last pass, but can you change the identifier to conform to the `underscore_lowercase` convention that we use elsewhere, for consistency?